lib/taskana-core/src/main/java/pro/taskana/workbasket/api/WorkbasketService.java

```diff
@@ -250,11 +258,12 @@ void addDistributionTarget(String sourceWorkbasketId, String targetWorkbasketId)
    *     Workbasket}
    * @param targetWorkbasketId The {@linkplain Workbasket#getId() id} of the target {@linkplain
    *     Workbasket}
-   * @throws NotAuthorizedException If the current user doesn't have {@linkplain
+   * @throws MismatchedWorkbasketPermissionException If the current user doesn't have {@linkplain
    *     WorkbasketPermission#READ READ permission} for the source {@linkplain Workbasket}
+   * @throws MismatchedRoleException If the current user doesn't have correct role
```

Comment:
if the current user is not member of {@linkplain TaskanaRole#BUSINESS_ADMIN} or {@linkplain TaskanaRole#ADMIN}